share/completions/flatpak.fish

```diff
@@ -30,9 +31,21 @@ complete -f -c flatpak -n "not __fish_seen_subcommand_from $commands" -a build-i
 complete -f -c flatpak -n "not __fish_seen_subcommand_from $commands" -a build-sign -d 'Sign an application or runtime'
 complete -f -c flatpak -n "not __fish_seen_subcommand_from $commands" -a build-update-repo -d 'Update the summary file in a repository'
 
-complete -f -c flatpak -n "__fish_seen_subcommand_from run" -a "(flatpak list --app | string match -r '\S+')"
-complete -f -c flatpak -n "__fish_seen_subcommand_from info uninstall" -a "(flatpak list | string match -r '\S+')"
-complete -f -c flatpak -n "__fish_seen_subcommand_from remote-ls remote-modify remote-delete" -a "(flatpak remote-list | string match -r '\S+')"
+if test (string join . "$flatversion[1]" "$flatversion[2]") -ge 1.2
```

Comment:
This works because we force the c locale for float comparisons, otherwise you'd have issues with locales that use "," as a decimal separator like e.g. german.

But it does work, and we're not going to change this, so it's okay. It's just an idea to keep in mind, because other programs might not be so accomodating.

I would however ignore stderr here - `2>/dev/null`, in case we didn't get a version.

Reply:
> I would however ignore stderr here - `2>/dev/null`, in case we didn't get a version.

This will make completions fallback to v1.0. Sounds good.